docs/blog/Introducing.md

```diff
@@ -0,0 +1,118 @@
+
+This post introduces the main artifacts and their features.
+
+## AI Orchestration
+
+```xml
+<dependency>
+  <groupId>com.sap.ai.sdk</groupId>
+  <artifactId>orchestration</artifactId>
+  <version>1.0.0</version>
+</dependency>
+```
+
+Use the generative AI Hub orchestration service with our `orchestration` module to set up templating, content filtering, and data masking for your applications.
+The orchestration service streamlines AI interactions while ensuring adherence to content safety guidelines.
+
+* Templating: Build dynamic prompts with placeholders to tailor AI interactions to user inputs.
+* Content Filtering: Apply filters to maintain compliance with content safety guidelines.
+* Data Masking: Anonymize and pseudonymize sensitive data.
+* Grounding:  Add external data sources for contextually relevant information (planned for Q1 2025).
+
+**Example:** Write a simple chat completion.
+
+```java
+var client = new OrchestrationClient();
+var config = new OrchestrationModuleConfig().withLlmConfig(OrchestrationAiModel.GPT_4O);
+var prompt = new OrchestrationPrompt("Hello world! Why is this phrase so famous?");
+var result = client.chatCompletion(prompt, config);
+
+String messageResult = result.getContent();
+```
+
+You can learn more about the orchestration service capabilities [here](https://github.com/SAP/ai-sdk-java/blob/main/docs/guides/ORCHESTRATION_CHAT_COMPLETION.md).
+
+## AI Management
+
+```xml
+<dependency>
+  <groupId>com.sap.ai.sdk</groupId>
+  <artifactId>core</artifactId>
+  <version>1.0.0</version>
+</dependency>
+```
+
+Automate tasks such as creating artifacts, configurations, and deployments, executing batch inference jobs, as well as managing Docker registries and object storage for training data.
+The `core` module provides tools for workflow and scenario management within SAP AI Core.
+* Artifact Management: Register and organize datasets and model artifacts.
+* Configuration Management: Set up configurations for various models and use cases.
+* Deployment Management: Deploy AI models and manage their lifecycle within SAP AI Core.
+
+**Example:** Create a deployment in SAP AI Core.
+
+```java
+var api = new DeploymentApi();
+var resourceGroupId = "default";
+var request =
+  AiDeploymentCreationRequest.create().configurationId("12345-123-123-123-123456abcdefg");
+
+AiDeploymentCreationResponse deployment = api.create(resourceGroupId, request);
+String id = deployment.getId();
+AiExecutionStatus status = deployment.getStatus();
+```
+
+You can learn more about the SAP AI Core capabilities [here](https://github.com/SAP/ai-sdk-java/blob/main/docs/guides/AI_CORE_DEPLOYMENT.md).
+
+## Generative AI with Foundation Models and OpenAI
```

Comment:
Why OpenAI specifically?
Pardon my ignorance, if other models are available too, then mention that OpenAI is taken only as an example. Also add a link to a list of all models available in the SDK.

If it is only OpenAI for now, then state as such, adding that others will be added later (if they would be).

Reply:
@jjtang1985 is it okay to phrase the limitation and outlook [like that..](https://github.com/SAP/ai-sdk-java/pull/154/files/9523843c656d3d215aca8a12a5ba65f20d853f00..fee161f46d8b234f179a0ce04cacfeade62e5dea)?

> Currently only `openai` is supported.
> We are planning support for `vertex-ai` and `anthropic` soon - depending on user feedback.

Reply:
In that case, also add how to add feedback - that is pointer to create an issue (if you are inviting for it already). Otherwise the reader will wonder - where to give feedback.

Reply:
I would only add the following.
> Currently only openai is supported.
> Please open a [feature request](https://github.com/SAP/ai-sdk-java/issues/new/choose), if you need direct LLM access for other foundation models.